man/deepsea-customization.7

```diff
@@ -63,7 +63,9 @@ will now load your custom.sls.
 .SH More examples
 
 
-DeepSea ships a number of non-default states that can be used. For example we offer alternating stage.0s which 
+The default state is update-no-reboot, 
```

Comment:
I'd suggest not including this line and revert to "DeepSea ships a number of non-default states..." because starting this section with "The default state is update-no-reboot" is only true for stage.0, but this man page seems to be describing general customization techniques